chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -91,7 +91,8 @@ def self.bridge?(nic)
   def self.ovs_bridge?(nic)
     nic.kind_of?(::Nic::OvsBridge) or
       # If no ovs tools are installed there's no ovs switch
-      if ::Kernel.system("which ovs-vsctl > /dev/null 2>&1")
+      if ::Kernel.system("which ovs-vsctl > /dev/null 2>&1 && " \
+                         "systemctl -q is-active openvswitch.service")
```

Comment:
BTW, there is an option to just stop ovs-vsctl from writing to syslog (which could be ok in this case):

```ovs-vsctl --verbose=vsctl:syslog:emer --verbose=reconnect:syslog:emer br-exists #{nic}```

Though I am undecided about which approach to prefer.